src/plugins/workspace/public/components/workspace_form/workspace_permission_setting_input.test.tsx

```diff
@@ -112,3 +113,20 @@ describe('WorkspacePermissionSettingInput', () => {
     expect(onDeleteMock).toHaveBeenCalledWith(0);
   });
 });
+
+it('should call onTypeChange with types after types changed', () => {
```

Comment:
Can we move this test case inside describe section?

Reply:
sorry I didn't notice that